modules/persistent-storage-csi-smb-cifs-limits.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-smb-cifs-limits_{context}"]
+= Limitations
+
+The following limitations apply to the Common Internet File System (CIFS)/Server Message Block (SMB) Container Storage Interface (CSI) Driver Operator:
+
+* FIPS mode is not supported: 
++
+When Federal Information Processing Standards (FIPS) mode is enabled, the use of md4 and md5 are disabled, which prevents users from using ntlm, ntlmv2, or ntlmssp authentication. Also, signing cannot be used since it uses md5. Any CIFS mount that uses these methods fails when FIPS mode is enabled.
```

Comment:
Per [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s), avoid using "since" in place of "because" for translatability purposes:
```suggestion
When Federal Information Processing Standards (FIPS) mode is enabled, the use of md4 and md5 are disabled, which prevents users from using ntlm, ntlmv2, or ntlmssp authentication. Also, signing cannot be used because it uses md5. Any CIFS mount that uses these methods fails when FIPS mode is enabled.
```